File: 3986-number-of-elapsed-seconds-between-two-times/3986-number-of-elapsed-seconds-between-two-times.java
class Solution {
    private int seconds (String str) {
        int h = 10 * (str.charAt(0) - '0') + (str.charAt(1) - '0');
        int m = 10 * (str.charAt(3) - '0') + (str.charAt(4) - '0');
        int s = 10 * (str.charAt(6) - '0') + (str.charAt(7) - '0');

        return 3600 * h + 60 * m + s;
    }
    public int secondsBetweenTimes(String st, String et) {
        return seconds(et) - seconds(st);
    }
}

File: 3986-number-of-elapsed-seconds-between-two-times/3986-number-of-elapsed-seconds-between-two-times.py
class Solution:
    def secondsBetweenTimes(self, startTime: str, endTime: str) -> int:
        s = startTime.split(':')
        e = endTime.split(':')
        hours = int(e[0]) - int(s[0])
        mins = int(e[1]) - int(s[1])
        seconds = int(e[2]) - int(s[2])

        return seconds + 60 * mins + 60 * 60 * hours